packages/jumpstarter-driver-network/jumpstarter_driver_network/streams/websocket.py

```
@@ -25,7 +25,8 @@ class WebsocketServerStream(ObjectStream[bytes]):
 
     ws: WSConnection = field(init=False, default_factory=lambda: WSConnection(ConnectionType.SERVER))
     queue: Tuple[MemoryObjectSendStream[bytes], MemoryObjectReceiveStream[bytes]] = field(
-        init=False, default_factory=lambda: create_memory_object_stream[bytes](32)
+        init=False,
+        default_factory=lambda: create_memory_object_stream[bytes](32),  # ty: ignore[call-non-callable]
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)